var/spack/repos/builtin/packages/libtool/package.py

```diff
@@ -3,7 +3,7 @@
 class Libtool(Package):
     """libtool -- library building part of autotools"""
     homepage = "https://www.gnu.org/software/libtool/"
-    url      = "http://ftpmirror.gnu.org/libtool/libtool-2.4.2.tar.gz"
+    url      = "http://ftpmirror.gnu.org/libtool/libtool-2.4.6.tar.gz"
```

Comment:
ps. as an example, the basic url for trilinos is `12.2.1` https://github.com/LLNL/spack/blob/develop/var/spack/repos/builtin/packages/trilinos/package.py#L17, there is a whole range of versions up to `12.6.1`.